Clother/Views/HomeView.swift

```diff
@@ -10,19 +10,60 @@ import SwiftUI
 struct HomeView: View {
     // TODO: this is the entry point of your app. Hence, you must do the following in this view:
     /*
-        - Instantiate the app's view models, which will also be shared with all other views.
-        - Build the home view UI, which includes the toggle between the 'Clothes' section and 'Outfits' section.
-        - Observe the UI components implemented under subviews folder. Do not reimplement a subview that has been built, use the subviews given to you as much as possible.
-        - Navigation should also work in this view. This screen can jump to multiple views as shown in Figma. Utilize the view model functions and variables for navigation. Since this is the root view, don't forget to wrap it with a navigation stack.
-        - Your view models should be injected into the navigation and subview hierarchies as an 'environment object'.
+     - Instantiate the app's view models, which will also be shared with all other views.
+     - Build the home view UI, which includes the toggle between the 'Clothes' section and 'Outfits' section.
+     - Observe the UI components implemented under subviews folder. Do not reimplement a subview that has been built, use the subviews given to you as much as possible.
+     - Navigation should also work in this view. This screen can jump to multiple views as shown in Figma. Utilize the view model functions and variables for navigation. Since this is the root view, don't forget to wrap it with a navigation stack.
+     - Your view models should be injected into the navigation and subview hierarchies as an 'environment object'.
      */
-
+    @State var displayItems: Bool = true
+    
+    @State private var selectedServiceIndex = 0
+    private let serviceOptions = ["Clothes", "Outfits"]
+    
     var body: some View {
-        Text("")
+        VStack {
+            Picker(selection: $selectedServiceIndex, label: Text("Hello")) {
```

Comment:
Make sure to implement the UI for this as in Figma. Also, please use the home view model for handling switching between Clothes and Outfits. It has most of the functionality you need to achieve that. Don't use displayItems to achieve that logic.